modules/apps/commerce/commerce-product-service/src/main/java/com/liferay/commerce/product/internal/permission/CommerceChannelPermissionImpl.java

```diff
@@ -133,7 +150,53 @@ private boolean _contains(
 			commerceChannel.getCommerceChannelId(), actionId);
 	}
 
+	private boolean _hasSupplierAccount(
```

Comment:
pls check how this stack up with https://github.com/liferay-commerce/liferay-portal/pull/3917

Reply:
#3917 needs to be updated to prevent the duplicate. The commits for COMMERCE-11163 in that PR were done using old commits. I left a comment on the PR.

Reply:
I communicated with @dsitu  on this. @luca-pellizzon , we should be able to resolve the commit duplicate/dependencies soon.